src/T2DMap.h

```diff
@@ -113,7 +113,6 @@ class T2DMap : public QWidget
     int mFontHeight;
     bool mShowRoomID;
     QMap<int, QPixmap> mPixMap;
-//    QMap<int, QPixmap > mGridPix;
```

Comment:
This will be required, I think, when we restore the currently absent ability to produce pictures of area maps {TLuaInterpreter::exportAreaImage(areaId)}.